scripts/uberenv/project.json

```diff
@@ -4,5 +4,8 @@
 "package_final_phase" : "hostconfig",
 "package_source_dir" : "../..",
 "spack_url": "https://github.com/spack/spack",
-"spack_commit": "d98b433a3c14af5aaba03fd439f6e47cb970bac6"
+"spack_commit": "d98b433a3c14af5aaba03fd439f6e47cb970bac6",
+"vcpkg_url": "https://github.com/microsoft/vcpkg",
+"vcpkg_commit": "b79f7675aaa82eb6c5a96ae764fb1ce379a9d5d6",
+"vcpkg_triplet": "x64-windows"
```

Comment:
@agcapps  -- I set the default ``vcpkg-triplet`` to ``x64-windows`` (instead of ``x86-windows``).  This implies that one has to use the 64-bit msvc compiler / CMake generator, e.g. using the ``--msvc 201764`` option in ``config-build``.

Reply:
Why?

Reply:
It was mostly because I wanted to check that something other than the default (`x86-windows`), works through `uberenv` and through `vcpkg`. 

We can change it back if you think that would be better.